assignments/prototypes.js

```diff
@@ -7,14 +7,23 @@
   
   Each class has unique properites and methods that are defined in their block comments below:
 */
-  
+
 /*
   === GameObject ===
   * createdAt
   * dimensions
   * destroy() // prototype method -> returns the string 'Object was removed from the game.'
 */
 
+function GameObject(el1) {
```

Comment:
I have to try putting an end to numbers in variable names early. We all do it sometimes, especially myself only for my trivial things or if there is a functional logic behind the string value in a key. So do as I say not as I do applies.
example:
``` javascript
const exampleObject{
  parent,
  children,
  value_1,
  value_2, ...
}
// you could then for loop over only the keys and also use the value_# <- (number)
// for additional logic, such as if the order matters largely
```
I hope this explains when to create an alias with a number in it.
For semantic value we reserve this for the situation above to programmaticly use the number.
However styles will vary and I don't want to be pedantic, I just thought you might appriciate some observations.